docs/usage.rst

```diff
@@ -0,0 +1,256 @@
+Usage
+-----
+REISE.jl can be used within the Julia interpreter or through the Python scripts.
+
+Julia
++++++
+After the installation (see ):doc:`installation`), the REISE package is registered and
```

Comment:
I wasn't able to get this to render fully, so I'm only checking the .rst files. I just want to make sure that the `installation` reference is right because it looks a little different in the .rst file.

Reply:
Good catch. I removed the reference since i) it is pretty obvious that we refer to the previous section and ii) I am not sure it is worth the pain to create a target for the section to refer to